### DISCUSSION TITLE
Changing the path where extensions are installed.

### DISCUSSION THREAD

**Discussion by vsjha18** [coder/code-server#2172] [Q&A] [answered]
I use the standalone version 3.5.0 because I don't have root permission on my server. code-server dumps all the extensions and data files in this location `~/.local/share/code-server/`. The problem is in my org home directories are very limited in size. Can I change this location to something else using some configuration option ?

**Comment by code-asher** [comment, marked as answer]
Yup, check out `code-server --help`. To change this directory you can
use `--user-data-dir`.

**Reply by vsjha18** [reply]
This has worked !! Solved a big problem for thanks !!
